tests/downloaders/test_usenix.py

```diff
@@ -43,10 +44,11 @@ def test_usenix_fetch(tmp_config: TemporaryConfiguration,
                       monkeypatch: MonkeyPatch,
                       url: str) -> None:
     cls = papis.downloaders.get_downloader_by_name("usenix")
-    assert cls is Downloader
+    assert cls is papis.downloaders.usenix.Downloader
```

Comment:
Why is this needed? `Downloader` should already be imported from `usenix`, no?

Reply:
Had some trouble spelling out the check -- in some cases, mypy was still inferring the base class. But indeed, just replacing it with `Downloader` works.